src/protocols/aeternity/views/TransferSendModal.vue

```diff
@@ -89,16 +95,30 @@ export default defineComponent({
       && isUrlValid(transferData.value.address)
     ));
 
-    const isSendingDisabled = computed(() => (
-      error.value
-      || !isAeNodeReady.value
-      || !transferData.value.address
-      || !transferData.value.amount
+    const showNextButton = computed(() => (
+      currentStep.value === TRANSFER_SEND_STEPS.review
+      && props.isAirGap
     ));
 
-    const customPrimaryButtonText = computed(() => (props.isMultisig)
-      ? t('modals.multisigTxProposal.proposeAndApprove')
-      : '');
+    const isSendingDisabled = computed(() => {
+      if (showNextButton.value) {
+        return (error.value || !isAeNodeReady.value);
+      }
+      return error.value
+        || !isAeNodeReady.value
+        || !transferData.value.address
+        || !transferData.value.amount;
```

Comment:
In my opinion this is really hard to read now. Maybe there is a way to simplify this?
The name of the computed property is related to "sending". So having `showNextButton` here creates confusion.